src/config.hpp

```diff
@@ -98,6 +98,16 @@ class Config {
         return result->operator[]("port").as<uint64_t>();
     }
 
+    /**
+         * @brief Get the gRPC network interface address to bind to
+         * 
+         * @return const std::string&
+         */
+    const std::string grpcBindAddress() {
```

Comment:
new command options should be documented in readme files.

Reply:
done